src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,44 @@
+package duke.task;
```

Comment:
Event and Deadline share a lot of similarities, perhaps it will make it simpler if they Deadline inherits from Event?